lib/server/controller.js

```diff
@@ -701,6 +701,11 @@ exports.getWindowSize = function (req, res) {
   req.device.getWindowSize(windowHandle, getResponseHandler(req, res));
 };
 
+exports.maximizeWindow = function (req, res) {
+  // noop
+  res.sendStatus(200);
```

Comment:
probably want to do something like `respondSuccess(req, res);` here instead to take care of jsonwp-ing everything